lib/manageiq/cross_repo/runner.rb

```diff
@@ -146,7 +147,7 @@ def load_travis_yml!
       # Set missing travis sections to the proper defaults
       travis_yml["install"] ||= travis_defaults[travis_yml["language"]]["install"]
 
-      travis_yml["script"] = script_cmd if script_cmd.present?
+      travis_yml["script"] = script_cmd unless script_cmd.nil?
```

Comment:
reason for the `nil` check rather than just treating the variable as a boolean?

Maybe it was chessbyte that liked to explicitly state `nil?`